neuralop/layers/fno_block.py

```diff
@@ -146,34 +144,28 @@ def __init__(
         self.ada_in_features = ada_in_features
 
         # apply real nonlin if data is real, otherwise CGELU
-        if complex_data:
+        if self.complex_data:
             self.non_linearity = CGELU
         else:
             self.non_linearity = non_linearity
-    
-        # TODO: eventually support complex data in SphericalConv and SFNO
-        # remove these lines once support is added
-        if conv_module == SpectralConv:
-            complex_kwarg = {'complex_data': complex_data}
-        else:
-            complex_kwarg = dict()
-
-        self.convs = conv_module(
-            self.in_channels,
-            self.out_channels,
-            self.n_modes,
-            output_scaling_factor=output_scaling_factor,
-            max_n_modes=max_n_modes,
-            rank=rank,
-            fixed_rank_modes=fixed_rank_modes,
-            implementation=implementation,
-            separable=separable,
-            factorization=factorization,
-            decomposition_kwargs=decomposition_kwargs,
-            joint_factorization=joint_factorization,
-            n_layers=n_layers,
-            **complex_kwarg
-        )
+        
+        self.convs = nn.ModuleList([
```

Comment:
Could it be better to name the self.convs as self.spectral_conv? I feel people might get confused it with CNN.